### PR TITLE
editing the navigation display bar to correct css

### PR DESCRIPTION
**Pull request by dubdabasoduba**

### CODE REVIEW

File: omod_1.x/src/main/webapp/bill.jsp [hunk 1]
```diff
@@ -17,8 +16,8 @@
 <openmrs:require allPrivileges="<%=PrivilegeWebConstants.BILL_PAGE_PRIVILEDGES%>" otherwise="/login.htm" redirect="/module/openhmis/cashier/bill.form" />
 
 <%-- Force our newer jquery version to load first --%>
-<openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/lib/jquery.js" />
 <%@ include file="/WEB-INF/template/header.jsp"%>
+<openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/lib/jquery.js" />
```
Comment (akmad) [inline omod_1.x/src/main/webapp/bill.jsp]: Isn't this change going to reverse the fix making our version of jquery load first?

Comment (dubdabasoduba) [inline omod_1.x/src/main/webapp/bill.jsp]: i tested it and it works fine and does not break any jquery functionality. The older jquery is loaded when the localheader is included isn't??

Comment (akmad) [inline omod_1.x/src/main/webapp/bill.jsp]: What version of OpenMRS did you test with? This was an issue when running older versions of OpenMRS that included a very old version of jquery.

Comment (dubdabasoduba) [inline omod_1.x/src/main/webapp/bill.jsp]: I tested it using the 2.x version. Let me do a test with the older version then i will let you know what happens

**Pull request merged**